readme.md

```diff
@@ -96,6 +96,11 @@ prompt pure
 
 ## Colors
 
+As explained in ZSH's[manual](http://zsh.sourceforge.net/Doc/Release/Zsh-Line-Editor.html#Character-Highlighting) color values can be:
+- a decimal integer up to the maximal number of color your terminal support
```

Comment:
What does the decimal integer represent? How can users find the number they want? Those are some questions we should answer.

In general, we should not force users to read the horrible ZSH manual to achieve things most user would want.

Reply:
I have explained what it represent but the way to find the which color that I can't explain because it is terminal specific.

Now I understand the reason behind all your comments on the documentation side. And I'm happy to see I'm not the only one thinking that this manual isn't well written! Hell I have even found this gem in the manual:
> [...] the Zsh Development Group accepts *no responsibility* for any brain damage which may occur during the reading [...]